catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -360,4 +363,8 @@ private void fileProcessingComplete(String resource) {
       LOGGER.debug("Unable to create {} to indicate {} processed", processedIndicator, resource, e);
     }
   }
+
+  private long getSortPopVal(long popSize) {
+    return Math.min(10, (popSize - 1) / 50_000 + 2);
```

Comment:
Bunch of random constants here, I would pull them out and label them.

Reply:
maybe a comment on the method would be sufficient